packages/core/src/amazonqFeatureDev/util/files.ts

```diff
@@ -28,17 +29,17 @@ export async function prepareRepoData(
     repoRootPaths: string[],
     workspaceFolders: CurrentWsFolders,
     telemetry: TelemetryHelper,
-    span: Span<AmazonqCreateUpload>
+    span: Span<AmazonqCreateUpload>,
+    zip: AdmZip = new AdmZip()
```

Comment:
FYI: AdmZip will be replaced by zip.js https://github.com/aws/aws-toolkit-vscode/pull/4769

Reply:
ah, I see. Once that PR is merged I can refactor. I'll add a note to the description.

Reply:
Went ahead and deleted the AdmZip work. Once a new zip is implemented, this can be augmented to spy on it as needed.

Reply:
https://github.com/aws/aws-toolkit-vscode/pull/4769 is merged now